src/palette/view/widgets/palettewidget.cpp

```diff
@@ -846,7 +846,12 @@ void PaletteWidget::dragMoveEvent(QDragMoveEvent* event)
 {
     if (event->source() == this) {
         if (m_currentIdx != -1 && event->proposedAction() == Qt::MoveAction) {
-            int targetIdx = cellIndexForPoint(event->pos());
+#ifdef MU_QT5_COMPAT
+            QPoint pos = event->pos();
```

Comment:
This one also should be QPointF (same for the other case in this file)

Reply:
This `pos` is passed to a method that receives `QPoint`
```
int PaletteWidget::cellIndexForPoint(const QPoint& p) const
```

Reply:
Ah, for me that method takes QPointF, but that turns out to be because I adjusted it myself!

Reply:
I usually prefer not to mix different tasks. For example, in this case, for me there is a separate task of “porting” and a separate task of “refactoring” the use of a data type. In my experience, mixing often leads to complexity and problems :)